src/helpers.js

```diff
@@ -2,16 +2,40 @@ import MutationObserver from '@sheerun/mutationobserver-shim'
 
 const globalObj = typeof window === 'undefined' ? global : window
 
+function runWithRealTimers(callback) {
```

Comment:
Let's add a comment right above here indicating that currently we only support jest but could support other frameworks in the future.

Reply:
Done :)